eng/helix/helix.proj

```diff
@@ -60,17 +46,15 @@
   <ItemGroup>
     <!--
       Use the BrowserDebugHost transport package as a sentinel for the non-shipping version of the NETCoreApp shared framework.
-      !temporary! e.g. https://dotnetbuilds.blob.core.windows.net/internal/Runtime/6.0.2-servicing.22060.12/dotnet-runtime-6.0.2-win-x86.zip?{token}
     -->
-    <HelixCorrelationPayload Include="dotnet-cli"
-        Condition="'$(SYSTEM_TEAMPROJECT)' == 'internal' AND '$(IsPublicRuntime)' == 'false'"
-        Destination="dotnet-cli"
-        Uri="https://dotnetbuilds.blob.core.windows.net/internal/Runtime/$(MicrosoftNETCoreBrowserDebugHostTransportVersion)/dotnet-runtime-$(MicrosoftNETCoreAppRuntimeVersion)-$(DotNetCliRuntime)$(ArchiveExtension)$([System.Environment]::GetEnvironmentVariable('DotNetBuildsInternalReadSasToken'))" />
-
-    <AdditionalDotNetPackage Include="$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
-                             Condition="'$(SYSTEM_TEAMPROJECT)' != 'internal' OR '$(IsPublicRuntime)' == 'true'">
+    <AdditionalDotNetPackage Include="$(MicrosoftNETCoreBrowserDebugHostTransportVersion)">
       <PackageType>runtime</PackageType>
     </AdditionalDotNetPackage>
+    
+    <AdditionalDotNetPackageFeed Include="https://dotnetbuilds.blob.core.windows.net/internal"
```

Comment:
Let's confirm this feed is correct and that we need to duplicate this code in multiple repos before merging this. That is, we should really not need to add an `@(AdditionalDotNetPackageFeed)` item though the rest is great to see.

Not sure the branch is currently open anyhow.

Reply:
@mmitche @MattGal should we use this feed of the `azureedge` one?

I don't think we need this workaround anywhere else. As far as I know, we're the only repo that runs helix tests against the incoming dotnet/aspnet/windowsdesktop version - if there was another such repo, they would've needed a workaround similar to the one we're replacing.

What other confirmation do you need that we need `AdditionalDotNetPackageFeed`? 7.0 fails without it & arcade doesn't include a reference to this feed

Reply:
Yeah, I can't find an instance of `AdditionalDotNetPackage` in any of the other repo in the stack

Reply:
For internal, use the blob.core.windows.net one. The CDN doesn't cache anything with a query string.

Reply:
> What other confirmation do you need

Sorry, I wasn't asking for confirmation about the requirement. My question was much more about whether everything but the `env:` changes should be in the Arcade repo. Since we've got the hook and are the only Arcadified repo needing this stuff in our Helix correlation payload, I'm fine w/ making the change here.